### DISCUSSION TITLE
Hero 7 Bitrate

### DISCUSSION THREAD

**Discussion by HalideMedia** [gopro/labs#114] [Q&A] [answered]
Is there a way to set the Hero 7 Black bitrate using GoPro Labs advanced firmware and the QRControl app?  Or is that only for Hero 10 and 11?  Are we still locked in to a 78 megabit per second maximum?

**Comment by dnewman-gpsw** [comment, marked as answer]
The bitrate hack came later, plus it needed GP2 to perform reliably.  Same reason there are no bitrate extensions for H8, H9 and MAX.
